cfme/infrastructure/provider/__init__.py

```diff
@@ -171,7 +171,7 @@ def __init__(
         self.key = key
         self.provider_data = provider_data
         self.zone = zone
-        self.vm_name = version.pick({version.LOWEST: "VMs", '5.5': "VMs and Instances"})
+        self.vm_name = deferred_verpick({version.LOWEST: "VMs", '5.5': "VMs and Instances"})
```

Comment:
This change is necessary and works fine. Without this, you are unable to list providers from yamls without having access to an appliance.